features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,254 @@
+    | 6014      | Verenigde Staten van Amerika |             |                                  |                             | 20181201 | 20200415 |
+    |           |                              |             |                                  | Woonboot bij de Grote Sloot | 20160526 | 20181201 |
+    |           |                              | Laan        | 0800010000000001                 |                             | 20100818 | 20160526 |
+
+Rule: De gevraagde velden van een verblijfplaats van een persoon met deels/geheel onbekende datum aanvang adreshouding/adres buitenland worden geleverd als de onzekerheidsperiode van de adreshouding van de verblijfplaats in de gevraagde periode ligt of als de gevraagde periode de onzekerheidsperiode geheel/deels overlapt
+
+  De adreshouding van een persoon heeft een onzekerheidsperiode als de datum aanvang adreshouding geheel/deels onbekend is.
+  In deze periode kan niet met zekerheid worden aangegeven of de persoon daadwerkelijk bewoner is van het adresseerbaar object.
+  Voor een datum aanvang adreshouding waarvan de dag onbekend is, loopt de onzekerheidsperiode vanaf de eerste dag tot en met de laatste dag van de betreffende maand.
+  Voor een datum aanvang adreshouding waarvan de dag en maand onbekend is, loopt de onzekerheidsperiode vanaf de eerste dag tot en met de laatste dag van het betreffende jaar.
+
+  De verblijfplaatsen worden aflopend gesorteerd geleverd. De eerste verblijfplaats in de verblijfplaatsen lijst is de meest recente verblijfplaats in de gevraagde periode.
+  In tegenstelling tot verblijfplaatsen met bekend datum aanvang adreshouding/adres buitenland, is de sortering niet hetzelfde als aflopend sorteren op datum aanvang adreshouding/adres buitenland.
+  Bij aflopende sortering zou een verblijfplaats met geheel onbekend datum aanvang adreshouding altijd de laatste verblijfplaats zijn in de verblijfplaatsen lijst.
+
+  Abstract Scenario: <scenario> (geen vorige en geen volgende adreshoudingperiode)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | <datum aanvang adreshouding>       |
+    Als gba verblijfplaatsen wordt gezocht met met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | <datum van>         |
+    | datumTot            | <datum tot>         |
+    | fields              | verblijfplaats      |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat | datumVan | datumTot |
+    | 0800010000000001                 | Laan   | 20100818 |          |
```

Comment:
moet deze datumVan zijn ingevuld? IOk weet niet of ik het snap....:-).

Reply:
Deze datum waarde heb ik over het hoofd gezien. Hier moet een placeholder staan, zodat de correcte datum aanvang adreshouding wordt toegevoegd